students/preactive/Lesson3/slice.py

```diff
@@ -0,0 +1,86 @@
+def slicer():
+    a_string = "this is a string"
+    a_tuple = (2, 54, 13, 12, 5, 32)
+    a_arr = ["!", "!", "tide", "pod", "challenge", "is",
+             "not", "for", "the", "weak", "!", "!"]
+    three_types = (a_string, a_tuple, a_arr)
+
+    for slice_item in three_types:
+        for_type = type(slice_item)
+        first_last(slice_item, for_type)
+        odd_out(slice_item, for_type)
+        ambigus_ao_fourth(slice_item, for_type)
+        reversed(slice_item, for_type)
```

Comment:
be careful about overwriting built-in functions, as you can see github highlights `reversed`, so will some code editors.

Reply:
Fixed, changed verbage